test/test_tsubaiso_sdk.rb

```diff
@@ -501,26 +502,43 @@ def test_list_sales
     @api.destroy_sale("AR#{september_sale[:json][:id]}") if september_sale[:json][:id]
   end
 
-  def test_list_sale_balances
-    first_sale = @api.create_sale(@sale_201608)
-    ar_segment = @api.show_customer(first_sale[:json][:customer_master_id])[:json][:used_in_ar]
-    realization_timestamp = Time.parse(first_sale[:json][:realization_timestamp])
-
-    customer_master_id = first_sale[:json][:customer_master_id]
-
-    balance_list_before = @api.list_sale_balances(realization_timestamp.year, realization_timestamp.month)
-    assert_equal 200, balance_list_before[:status].to_i, balance_list_before.inspect
-    assert(balance_list_before[:json].count > 0)
-
-    second_sale = @api.create_sale(@sale_201609)
-
-    balance_list_after = @api.list_sale_balances(realization_timestamp.year, realization_timestamp.month, :customer_master_id => customer_master_id, :ar_segment => ar_segment)
-    assert_equal 200, balance_list_after[:status].to_i, balance_list_after.inspect
-    assert(balance_list_after[:json] == balance_list_before[:json])
-
-  ensure
-    @api.destroy_sale("AR#{first_sale[:json][:id]}") if first_sale[:json][:id]
-    @api.destroy_sale("AR#{second_sale[:json][:id]}") if second_sale[:json][:id]
+  def test_list_sales_and_account_balances
+    begin
+      realization_timestamp = Time.parse(@sale_201608[:realization_timestamp])
+
+      # Without customer_master_id and ar_segment option parameters
+      balance_list_before = @api.list_sales_and_account_balances(realization_timestamp.year, realization_timestamp.month)
+      assert_equal 200, balance_list_before[:status].to_i, balance_list_before.inspect
+      assert(balance_list_before[:json].count > 0)
```

Comment:
507-512 test will have big possibilities test failure for another company, because it's rely on our fixtures. big possibility in production sale_201608 not belongs to another companies. so assert(balance_list_before[:json].count > 0) will fail. make new sale earlier is solve this problem I think.